ee/clickhouse/views/groups.py

```diff
@@ -187,7 +187,7 @@ def property_definitions(self, request: request.Request, **kw):
 
         group_type_index_to_properties = defaultdict(list)
         for group_type_index, key, count in rows:
-            group_type_index_to_properties[group_type_index].append({"name": key, "count": count})
+            group_type_index_to_properties[str(group_type_index)].append({"name": key, "count": count})
```

Comment:
Not sure I understand this change 🤔

Reply:
They became string keys anyway with the previous JSON serializer. This one threw an error, unless we flipped on option. Since we had it as string, I explicitly made it string so the serializer does not complain.
Sorry, could have mentioned in changes.